tests/tuned/etc/tuned/basic_settings/tuned_change.conf

```diff
@@ -0,0 +1,18 @@
+# tuned configuration
```

Comment:
The intention is that `tests/tuned/etc/tuned/` mirrors exactly what would be under `/etc/tuned` in a real system, so that if you want a valid `/etc/tuned` for testing, you can just copy `tests/tuned/etc/tuned/`.  Please correct me if I'm wrong, but from what I understand about tuned, each profile has its own named directory under `/etc/tuned/`, and each profile directory has one single config file named `tuned.conf` for that profile.  I think a profile directory may contain other files like scripts, but only 1 config file.  So having `tests/tuned/etc/tuned/basic_settings/` with both `tuned.conf` and `tuned_change.conf` breaks that model.  
Therefore, I suggest you create a new profile directory `tests/tuned/etc/tuned/change_settings/` and move `tuned_change.conf` to `tests/tuned/etc/tuned/change_settings/tuned.conf`

I apologize that my comment https://github.com/linux-system-roles/kernel_settings/pull/7#discussion_r440868873 was not clear, and that I did not explain the purpose and contents of `tests/etc/tuned/`